### DISCUSSION TITLE
Can you tell me if tailwindcss does not provide dist files in the v3 version?

### DISCUSSION THREAD

**Discussion by Yakumo-Yukari** [tailwindlabs/tailwindcss#11218] [Help]
https://unpkg.com/browse/tailwindcss@3.3.2/

Before version 3.0 you could download the css files in dist for local use, but in V3 there is no way to download them

Can I still download the v3 css elsewhere to use in my local html? I don't really want to use the cdn method to import

**Comment by wongjn** [comment]
Nope!
